backend/routes/login.py

```diff
@@ -0,0 +1,82 @@
+from fastapi import APIRouter, Body, HTTPException
+#from backend.models import user as user_
+from models.user import LoginUserSchema,VerifyOTPResponse,SessionToken
+from fastapi.encoders import jsonable_encoder
+from config.database import db as database
+from config.jwt_handler import JWT_ALGORITHM, JWT_SECRET
+from config.twilio_config import twilio_client, twilio_number
+from schemas.user import serializeDict, serializeList
+from bson import ObjectId
+import random
+from datetime import datetime,timedelta
+import time
+import jwt
+
+#app = FastAPI()
+router = APIRouter()
+
+
+
+def otp_generate_save(phone_number):
+    # Generate 4 digit random OTP and timestamp
+    otp = random.randint(1000, 9999)
+    timestamp = datetime.now()
+
+    # Upsert the phone_number:otp mapping
+    database.otp_mapping.update_one(
+        {'phone_number': phone_number},
+        {"$set": {'otp': otp, 'timestamp': timestamp}},
+        upsert=True
+    )
+
+    return otp
```

Comment:
please use already existing function in user file and import `otp_generate_save` function from there in this file.